internal/isoeditor/rhcos.go

```diff
@@ -80,6 +82,12 @@ func (e *rhcosEditor) CreateClusterMinimalISO(ignition string) (string, error) {
 		return "", err
 	}
 
+	if staticIPConfig != "" {
+		if err := e.addCustomRAMDisk(staticIPConfig); err != nil {
+			return "", err
```

Comment:
maybe worth some 'log.WithError' here

Reply:
We log in the caller so I'd prefer wrapping the error if you think we need more info.